src/uu/fmt/src/fmt.rs

```diff
@@ -69,6 +92,7 @@ pub struct FmtOptions {
 /// # Returns
 ///
 /// A tuple containing a vector of file names and a `FmtOptions` struct.
+#[allow(clippy::field_reassign_with_default)]
```

Comment:
Why not just follow the suggestion of the lint to this?
```rust
let mut fmt_opts = FmtOptions {
    tagged: matches.get_flag(OPT_TAGGED_PARAGRAPH),
    .. Default::default()
};
```

Reply:
I didn't follow the suggestion for consistency reasons. With the suggestion, the `fmt_opts.tagged` property would be set in a different way than all other properties, which are set with `fmt_opts.property_name = ...`

Reply:
Oh I see. Github wasn't showing me the other lines. Makes sense :)